### PR TITLE
Fix lakectl table write to non tty non string column

### LINKED ISSUE #5433: Output of `lakectl auth groups policies list` includes erroneous `%!s(int=0)`
When running `lakectl auth groups policies list` the results include `%!s(int=0)` which is a bug. 

```
$ lakectl -v

lakectl version 0.96.1

$ lakectl auth groups policies list --id "Data Scientists" 

AuthManageOwnCredentials	0001-01-01 00:00:00 +0000 UTC	%!s(int=0)	arn:lakefs:auth:::user/${user}	allow	auth:CreateCredentials, auth:DeleteCredentials, auth:ListCredentials, auth:ReadCredentials
FSBlockAccessToSensitiveData	2023-03-02 10:57:48 +0000 UTC	%!s(int=0)	arn:lakefs:fs:::repository/drones03/object/sensitive*	deny	fs:*
FSBlockMergingToMain	2023-03-06 12:22:01 +0000 UTC	%!s(int=0)	arn:lakefs:fs:::repository/*/branch/main	deny	fs:CreateCommit
FSReadWriteAll	0001-01-01 00:00:00 +0000 UTC	%!s(int=0)	*	allow	fs:ListRepositories, fs:ReadRepository, fs:ReadCommit, fs:ListBranches, fs:ListTags, fs:ListObjects, fs:ReadObject, fs:WriteObject, fs:DeleteObject, fs:RevertBranch, fs:ReadBranch, fs:ReadTag, fs:CreateBranch, fs:CreateTag, fs:DeleteBranch, fs:DeleteTag, fs:CreateCommit
RepoManagementReadAll	0001-01-01 00:00:00 +0000 UTC	%!s(int=0)	*	allow	ci:Read*, retention:Get*, branches:Get*, fs:ReadConfig

### PR DESCRIPTION
**Pull request by nopcoder**
Fix #5433

### CODE REVIEW

File: cmd/lakectl/cmd/common_helpers.go [hunk 1]
```diff
@@ -151,7 +151,7 @@ func WriteTo(tpl string, data interface{}, w io.Writer) {
 			var b strings.Builder
 			for _, row := range tab.Rows {
 				for ic, cell := range row {
-					b.WriteString(fmt.Sprintf("%s", cell))
+					b.WriteString(fmt.Sprintf("%v", cell))
```
Comment (arielshaqed) [inline cmd/lakectl/cmd/common_helpers.go:154]: Can we use just fmt.Sprint? (Should we, given the danger of format during confusion?)

**Review by arielshaqed — APPROVED** [review]
(Not sure this is specifically related to tty output.)

Thanks!

File: cmd/lakectl/cmd/common_helpers.go (hunk 1 above)
Comment (nopcoder) [reply cmd/lakectl/cmd/common_helpers.go:154]: I'll split the format and write. The Sprintf will render a single cell value - but in order to concat full table content I'm using strings.Builder.

**Comment by nopcoder** [comment]
> (Not sure this is specifically related to tty output.)
> 
> Thanks!

The non-tty is related to the fact that we use this code while render the table when we identify that the output is not the terminal.

**Pull request merged**